data-prepper-api/src/test/java/org/opensearch/dataprepper/model/source/SourcePartitionTest.java

```diff
@@ -6,6 +6,8 @@
 package org.opensearch.dataprepper.model.source;
 
 import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.model.source.coordinator.PartitionIdentifier;
+import org.opensearch.dataprepper.model.source.coordinator.SourcePartition;
```

Comment:
Why do you have these imports?

Reply:
Not sure why these are here. I don't have them in my local branch. Will make sure they are gone on the next commit.